Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.12-slim
+LABEL maintainer="mxbely98@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
```

Comment:
It's a good practice to specify the target directory when using the `COPY` command. For example, `COPY . /app` and then set the working directory with `WORKDIR /app`.